### PR TITLE
Fix rebuilding the DBs if .env gets updated

### PR DESCRIPTION
**Pull request by vmarkovtsev**
Signed-off-by: Vadim Markovtsev <vadim@athenian.co>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -30,6 +30,7 @@ $(ENV_FILE):
 
 $(IO_DIR)/%.sqlite: $(ENV_FILE)
 	docker run --rm -e DB_DIR=/io -v$(IO_DIR):/io --env-file $(ENV_FILE) --entrypoint python3 $(IMAGE) /server/tests/gen_sqlite_db.py
+	@touch $@
```
Comment (dpordomingo) [inline Makefile]: The database is created wit `read-only` attributes, so `touch` fails.

Comment (vmarkovtsev) [reply Makefile]: WDYM? It works fine for me.

Comment (vmarkovtsev) [reply Makefile]: Is it about the owner user?

Comment (vmarkovtsev) [reply Makefile]: How about now?

Comment (dpordomingo) [reply Makefile]: It did! Many thanks!

**Pull request merged**